app/ante/eip712.go

```diff
@@ -52,9 +52,9 @@ func init() {
 // transactions, as defined by the presence of an ExtensionOptionsWeb3Tx extension.
 func NewLegacyCosmosAnteHandlerEip712(options HandlerOptions) sdk.AnteHandler {
 	return sdk.ChainAnteDecorators(
-		RejectMessagesDecorator{}, // reject MsgEthereumTxs
+		NewRejectMessagesDecorator(options.DisabledCosmosMsgs),
 		// disable the Msg types that cannot be included on an authz.MsgExec msgs field
-		NewAuthzLimiterDecorator(options.DisabledAuthzMsgs),
+		NewAuthzLimiterDecorator(options.DisabledCosmosMsgs),
```

Comment:
I think only need to reject authz message by this `AuthzLimiterDecorator` decorator, `RejectMessagesDecorator` is not needed.

`RejectMessagesDecorator` is about to reject Ethereum msgs from being executed without mandatory EVM ante handlers.
`AuthzLimiterDecorator` is about to reject all Authz creating vesting account msgs from being executed, nesting checked, plus MsgEthereumTx, through Cosmos-ante handle.

Checking in both decorator seems duplicated right?

Reply:
It's not duplicate AFAIU

* if we want to block messages (e.g `EthereumMsg`, `Vesting`), it needs to be added to the  `RejectMessagesDecorator`
* To avoid the blocked msgs being executed by `Authz` workaround, it needs to be blocked additionally in `AuthzLimiterDecorator`

Reply:
@mtsitrin why u do [this](https://github.com/dymensionxyz/ethermint/commit/5abdf47e22aefe74a45d30a634a388dfb86bc850)? Are you implementing the whitelist policy?

Reply:
@mtsitrin my mistake, you are right, ty for pointing out.

Reply:
> @mtsitrin why u do [this](https://github.com/dymensionxyz/ethermint/commit/5abdf47e22aefe74a45d30a634a388dfb86bc850)? Are you implementing the whitelist policy?

yeah
Anyway the ethermint's app.go is "demo" code.
the actual blocked defitinom is set on rollapp-evm's `app.go` and passed in `HandlerOptions`

Reply:
I forgot abt that either 😂